src/fullscreen/fullscreen-controller.js

```diff
@@ -86,7 +89,13 @@ class FullscreenController {
         } else {
           const videoElement: ?HTMLVideoElement = this._player.getVideoElement();
           if (videoElement && typeof videoElement.webkitEnterFullScreen === 'function') {
-            videoElement.webkitEnterFullScreen();
+            if (this._player.isInPictureInPicture()) {
+              // iOS < 13 (iPad) has an issue to enter to full screen from PiP
+              setTimeout(() => videoElement.webkitEnterFullScreen(), 1000);
```

Comment:
There is a reason for 1s or it could be in the next tick?
I mean exitPictureInPicture and then setTimeout with timer=0